_data/compose-cli/docker_compose_down.yaml

```diff
@@ -1,63 +1,62 @@
 command: docker compose down
 short: Stop and remove containers, networks
 long: |-
-    Stops containers and removes containers, networks, volumes, and images created by ``up`.
+  Stops containers and removes containers, networks, volumes, and images created by ``up`.
```

Comment:
```suggestion
  Stops containers and removes containers, networks, volumes, and images created by `up`.
```

Reply:
good catch; opened https://github.com/docker/compose/pull/8616, and updated this PR